### PR TITLE
Bitbucket: Track stream state by repo and fix linter errors

### PR DESCRIPTION
**Pull request by cjwooo**
## Description

Also add optional repositories param.

## Type of change
- [x] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: sources/bitbucket-source/src/bitbucket/bitbucket.ts [hunk 1]
```diff
@@ -253,7 +253,7 @@ export class Bitbucket {
   }
 
   @Memoize(
-    (workspace: string, repoSlug: string): string => `${workspace}${repoSlug}`
+    (workspace: string, repoSlug: string): string => `${workspace};${repoSlug}`
```
Comment (tovbinm) [inline sources/bitbucket-source/src/bitbucket/bitbucket.ts:256]: why `;` ?

File: sources/bitbucket-source/resources/spec.json [hunk 2]
```diff
@@ -45,6 +45,17 @@
...
+        "type": "array",
+        "items": {
+          "type": "string"
+        },
+        "title": "Repositories",
+        "description": "Names of your Bitbucket repositories in the format 'workspace/repo-slug'. If none are provided, data from all repositories for the specified workspaces will be pulled.",
+        "examples": [
+          "blaze-lib/blaze"
+        ]
+      },
```
Comment (tovbinm) [inline sources/bitbucket-source/resources/spec.json:58]: what if both `workspaces` and `repositories` list are provided?

Comment (cjwooo) [reply sources/bitbucket-source/resources/spec.json:58]: Workspaces is always required. Repositories is optional and will indicate only certain repository data (commits, PRs, etc) is to be pulled. Workspace data (users, repo metadata) is always pulled.

File: sources/bitbucket-source/src/bitbucket/bitbucket.ts (hunk 1 above)
Comment (cjwooo) [reply sources/bitbucket-source/src/bitbucket/bitbucket.ts:256]: Just putting a separator in between the parameters. Following examples in the memoize library.

**Review by tovbinm — APPROVED** [review]

**Comment by tovbinm** [comment]
I am getting annoyed by the constant version bumps. Perhaps we can automatically release from the main branch on every merge instead of bumping it in the code?

**Comment by cjwooo** [comment]
We could try that.

**Pull request merged**